src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,44 @@
+package duke.command;
+
+import duke.util.Keyword;
+import duke.util.Message;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+/**
+ * Handles the command for delete.
+ *
+ * @author marcuspeh
+ * @version A-JavaDoc
+ * @since 23 Aug 2021
+ */
+public class DeleteCommand implements Command {
+    /** Stores the message entered by the . */
+    private String message;
+
+    /** Constructor for duke.command.DeleteCommand.
+     *
+     * @param message Stores the message entered by the user.
+     */
+    public DeleteCommand(String message) {
+        this.message = message;
+    }
+
+    /**
+     * Adds a new task to the task list.
+     *
+     * @param taskList duke.main.TaskList to execute the command.
+     * @param ui To interact with the user.
+     * @return message to be used by either the graphic UI or command line UI.
+     */
+    @Override
+    public Message execute(TaskList taskList, Ui ui) {
+        try {
+            return taskList.deleteTask(Integer.parseInt(message.substring(Keyword.DELETE.length() + 1)));
```

Comment:
Hi! Do you want to consider adding line breaks here to improve readability?

Reply:
Hmm where do u recommend adding in the line break?